Build_&_Go_SDK_Proto_Run.YML

```
@@ -0,0 +1,125 @@
+        run: GOARCH=${{ matrix.go-arch }} COSMOS_BUILD_OPTIONS=v2,bls12381 make build
+      - name: Build with Secp_cgo
+        if: matrix.go-arch == 'amd64'
+        run: GOARCH=${{ matrix.go-arch }} COSMOS_BUILD_OPTIONS=v2,secp make build
+      - name: Build Cosmovisor
+        run: GOARCH=${{ matrix.go-arch }} make cosmovisor
+      - name: Build Confix
+        run: GOARCH=${{ matrix.go-arch }} make confix
+      - name: Build Hubl
+        run: GOARCH=${{ matrix.go-arch }} make hubl
+
+name: Build & Push SDK Proto Builder
```

yamllint (1.35.1)

[error] duplication of key "name" in mapping

(key-duplicates)